services/apps/alcs/src/portal/public/search/notice-of-intent/public-notice-of-intent-search-view.entity.ts

```diff
@@ -63,8 +64,10 @@ import { LinkedStatusType } from '../public-search.dto';
           qb
             .from(NoticeOfIntentDecision, 'decision_date')
             .select('MAX("date")', 'date')
+            .addSelect('outcome_code', 'outcome')
             .addSelect('notice_of_intent_uuid', 'notice_of_intent_uuid')
-            .groupBy('notice_of_intent_uuid'),
+            .groupBy('notice_of_intent_uuid')
+            .addGroupBy('outcome'),
```

Comment:
Adding a second group by clause will cause bugs. If a notice of intent decision has two different outcomes, grouping by both uuid and outcome will result in two rows, instead of the expected result of just the newest decision.